var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -333,6 +333,16 @@ def _microarch_target_args(self):
         # Get our build microarchitecture
         microarch = self.spec.target
 
+        # We need to detect whether the target supports SVE before the magic for
+        # loop below which would change the value of `microarch`.
```

Comment:
Could you mention on which line the value of `microarch` got changed?

Reply:
https://github.com/spack/spack/pull/42107#issue-2083921220

Reply:
I see what you meant now.
```
     for microarch in microarch.ancestors: 
```
overwrites `microarch`. Anyway, this loop is not good. The case not hitting break was not handled properly.
Thanks.

Reply:
Agreed, but I didn't feel confident enough to fix that 🙂